tooi/api/statuses.py

```diff
@@ -20,7 +20,7 @@ async def context(status_id: str) -> Response:
 
 async def post(
     status: str,
-    visibility: Visibility = Visibility.Public,
+    visibility: Visibility | str = Visibility.Public,
```

Comment:
while you're here, it might be worth cleaning this up; `api` shouldn't import things from `screen`, and `Visibility | str` also feel wrong.  i would suggest making this a plain `str` in the API code.